Makefile

```diff
@@ -25,6 +25,13 @@ CONTROLLER_GEN := go run $(MOD_FLAGS) ./vendor/sigs.k8s.io/controller-tools/cmd/
 YQ_INTERNAL := go run $(MOD_FLAGS) ./vendor/github.com/mikefarah/yq/v2/
 KUBEBUILDER_ASSETS := $(or $(or $(KUBEBUILDER_ASSETS),$(dir $(shell command -v kubebuilder))), /usr/local/kubebuilder/bin)
 export KUBEBUILDER_ASSETS
+KUBEBUILDER_ASSETS ?= /usr/local/kubebuilder/bin
```

Comment:
This can be removed, the definitions above should cover this.

Reply:
I will remove this. It got pulled in when I rebased.